configure.ac

```diff
@@ -1279,6 +1279,17 @@ AC_RUN_IFELSE(
   AC_MSG_RESULT(cross-compile)
 )
 
+AC_MSG_CHECKING(for restrict qualifier support)
+AC_COMPILE_IFELSE(
+	[AC_LANG_PROGRAM([], [void modifybuf(char *restrict x) { }])],
+	AC_MSG_RESULT(yes)
+	[HAVE_RESTRICT_QUALIFIER=1],
+	[HAVE_RESTRICT_QUALIFIER=0]
+	AC_MSG_RESULT(no)
+)
+AC_SUBST(HAVE_RESTRICT_QUALIFIER)
+
+AST_GCC_ATTRIBUTE(__restrict__)
```

Comment:
So, here's the deal.  You don't actually need any of this.
Apparently, the last time bootstrap.sh was run it was run on an older system which generated a configure script that didn't detect that the compiler supports -std=gnu11 but doesn't enable it by default.  If you just run bootstrap.sh again on your system and commit the results, when someone runs configure on an older system the next time, configure should automatically add the `-std=gnu11` flag to the CC variable in makeopts.  This will allow chan_iax2 to compile.

Reply:
Hmm... so you're saying that flag should always be available on all supported platforms and thus restrict support can implicitly be a hard requirement? Wasn't sure if there might be other compatibility issues.

Reply:
As far as I can tell, as long as the configure script was generated on a recent system,it'll enable -std=gnu11 on gcc's as far back as we care to support.